.azure-pipelines/azure-pipelines-linux.yml

```diff
@@ -13,6 +13,7 @@ jobs:
         UPLOAD_PACKAGES: 'True'
         DOCKER_IMAGE: quay.io/condaforge/linux-anvil-cos7-x86_64
   timeoutInMinutes: 360
+  variables: {}
```

Comment:
What is the `variables` `{}` for?

Reply:
that was part of the re-render, not sure